src/lib/metrics.js

```diff
@@ -177,12 +177,13 @@ const Metrics = {
   installSuccess() {
     if (!installTimer) {
       // Skip this event if there's no install timer in progress.
-      return;
```

Comment:
using the installSuccess metrics function to return a boolean that will toggle an Oboarding component

Reply:
Probably shouldn't try hooking into this - it's just a timer for reporting to metrics if the add-on got installed within a reasonable amount of time.